lab3/src/task6/Main.java

```diff
@@ -0,0 +1,406 @@
+        Baravelli baravelli1 = new Baravelli("grape", "Italy", 6.7f, 8.7f);
+        ChocAmor chocAmor1 = new ChocAmor("coffee", "France", 5.5f);
+        ChocAmor chocAmor2 = new ChocAmor("vanilla", "France", 5.5f);
+        this.candies = new CandyBox[] {lindt1, lindt2, lindt3, baravelli1, chocAmor1, chocAmor2};
+    }
+    // Add getter and setter
+
+    public CandyBox[] getCandies() {
+        return candies;
+    }
+
+    public void setCandies(CandyBox[] candies) {
+        this.candies = candies;
+    }
+}
+
+class Area {
+    // TODO[0]: Add fields
+
+    private CandyBag candyBag;
+
+    private int number;
+
+    private String street;
+
+    private String message;
+
+    // TODO[1]: Constructor without parameters
+
+    public Area() {
+        this.number = 0;
+        this.street = "nowhere";
+        this.message = "nowhere";
+        this.candyBag = null;
+    }
+
+    // TODO[2]: Constructor which initialize all the fields
+
+    public Area(int number, String street, String message, CandyBag candyBags) {
+        this.number = number;
+        this.street = street;
+        this.message = message;
+        this.candyBag = candyBags;
+    }
+
+    // TODO[3]: Add getBirthdayCard() method with instanceof
+
+    public void getBirthdayCard() {
+        System.out.println("Street " + this.street + ", number " + this.number);
+        System.out.println();
+        System.out.println(this.message);
+        System.out.println();
+        for (CandyBox i : candyBag.getCandies()) {
+            if (i instanceof Lindt) {
+                ((Lindt) i).printLindtDim();
+            } else if (i instanceof Baravelli) {
+                ((Baravelli)i).printBaravelliDim();
+            }
+            if (i instanceof ChocAmor) {
+                ((ChocAmor)i).printChocAmorDim();
+            }
+        }
+    }
+
+    // TODO[4]: Add getBirthdayCardv2() method without instanceof
+
+    public void getBirthdayCardv2() {
+        System.out.println("Street " + this.street + ", number " + this.number);
+        System.out.println();
+        System.out.println(this.message);
+        System.out.println();
+        for (CandyBox i : candyBag.getCandies()) {
+            i.printDim();
+        }
+    }
+
+    // TODO[0]: Add getters and setters
+
+
+    public CandyBag getCandyBag() {
+        return candyBag;
+    }
+
+    public void setCandyBag(CandyBag candyBag) {
+        this.candyBag = candyBag;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String getStreet() {
+        return street;
+    }
+
+    public void setStreet(String street) {
+        this.street = street;
+    }
+
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+}
+
+public class Main {

```

Comment:
Don't put all the classes in a single file. When you push to git, I want to see how you structure your solution, and putting all the classes in a single file is not a good practice